Include/internal/pycore_global_strings.h

```diff
@@ -797,6 +797,11 @@ struct _Py_global_strings {
      ? (PyObject*)&_Py_SINGLETON(strings).ascii[(CH)] \
      : (PyObject*)&_Py_SINGLETON(strings).latin1[(CH) - 128])
 
+#define _Py_DECLARE_EMPTY_STRING    _Py_DECLARE_STR(empty, "")
```

Comment:
I don't think that _Py_DECLARE_EMPTY_STRING makes sense. Keeping _Py_DECLARE_STR() in unicode_get_empty() should be enough.

If you replace _Py_STR(empty) with _Py_EMPTY_STRING, you should remove _Py_DECLARE_STR(empty, ...) in all places except in unicode_get_empty().

Reply:
Sure! I'll make a separate commit for this in a moment.

Reply:
Actually `compile.c` also uses the macro in a function. The comment for `_Py_DECLARE_STR` says that it should be used before any invokation of `_Py_EMPTY_STRING` in a *function*. So, technically, we should also keep it (I'll also check that `_Py_EMPTY_STRING` is correctly paired for detection actually).